R/init.R

```diff
@@ -153,6 +153,21 @@ init <- function(data,
   # log
   teal.logger::log_system_info()
 
+  # invoke lockfile creation
+  # If user has setup the file, there is no need to compute a new one.
+  user_lockfile <- getOption("teal.renv.lockfile", "")
```

Comment:
in case you don't know... (hehe yet again)
there is a vignette `vignettes/teal-options.Rmd` describing options used - please add an entry about this one

Reply:
totally :)

Reply:
784057c0b330cfd303c2df75934069546bd3e172